docs/guides/editor_features/ai_completion.md

```diff
@@ -16,8 +16,14 @@ _Get started with Copilot_:
 1. Install [Node.js](https://nodejs.org/en/download).
 2. Enable Copilot via the settings menu in the marimo editor.
 
-_Note_: Copilot is not yet available in our conda distribution; please install
-marimo using `pip` if you need Copilot.
+```{admonition} Built from the ground up
+:class: note
+   marimo comes with GitHub Copilot, a tool that helps you write code faster by
+   suggesting in-line code suggestions based on the context of your current code.
+
+   marimo also comes with the ability to use AI for refactoring a cell, finishing writing a cell, or writing a full cell from scratch.
+   This feature is currently experimental and is not enabled by default.
+```
```

Comment:
@Haleshot , this appears to be duplicated at the top of the page.

Additionally, the admonition title seems to have been copied from a different title.